test/unit_tests/aievec_tests/i16xi16_static_sized_memref_aie-ml/conv2d_uij_i16_noinit.mlir

```diff
@@ -0,0 +1,70 @@
+// REQUIRES: valid_xchess_license
+// RUN: aie-opt %s -affine-super-vectorize="virtual-vector-size=16" -aieml=true --aie-vectorize="shift=10 zero-offset=4" | aie-translate -aieml=true --aievec-to-cpp -o gen_aie-ml.cc
```

Comment:
As a note: at some point we need to change this "floating option" (aieml) and make it a pass option for "aie-vectorize". Same in aie-translate, it should be an option for "aievec-to-cpp". In fact, we might want two translate passes, one for AIEv1 and one for AIEv2.

Reply:
Yes, we will do this after we finish the ops for aie2.